### PR TITLE
[FIX] runbot_send_email: Fixed github links

### PR DESCRIPTION
**Pull request by luiseevaquer**
[FIX] runbot_send_email: Fixed github links

### CODE REVIEW

File: runbot_send_email/models/runbot_build.py [hunk 1]
```diff
@@ -29,14 +33,28 @@ class RunbotBuild(models.Model):
     shareissue_link = fields.Char(compute='_shareissue_link')
 
     @api.multi
+    def get_github_links(self):
+        host_gh, owner_gh, repo_gh = False, False, False
```
Comment (moylop260) [inline runbot_send_email/models/runbot_build.py]: if you use `api.multi` use a `for record in self:` too

File: runbot_send_email/models/runbot_build.py [hunk 2]
```diff
@@ -29,14 +33,28 @@ class RunbotBuild(models.Model):
...
+    @api.multi
     def _host_name(self):
         base_url = self.env['ir.config_parameter'].get_param('web.base.url')
         self.host_name = urlparse(base_url).hostname
 
     @api.multi
     def _repo_name(self):
-        descrip = self.repo_id.name.replace('.git', '').replace(
-            'https://github.com/', '').replace('/', ' / ')
+        descrip = self.repo_link.replace('https://', '').replace('/', ' / ')
```
Comment (moylop260) [inline runbot_send_email/models/runbot_build.py]: Could you add a docstring with why this replace is needed?

Comment (luiseevaquer) [inline runbot_send_email/models/runbot_build.py]: at this time, only replace '/' for ' / ' to add space between owner and repo, this help to better read of the repo name

File: runbot_send_email/models/runbot_build.py [hunk 3]
```diff
@@ -29,85 +33,112 @@ class RunbotBuild(models.Model):
...
+                + record.name[:8]
 
     @api.multi
-    def _repo_name(self):
-        descrip = self.repo_id.name.replace('.git', '').replace(
-            'https://github.com/', '').replace('/', ' / ')
-        self.repo_name = descrip
+    def _host_name(self):
+        for record in self:
+            base_url = record.env['ir.config_parameter'].get_param(
```
Comment (moylop260) [inline runbot_send_email/models/runbot_build.py]: This value is even the same, then you can get it out of the `for`

File: runbot_send_email/models/runbot_build.py [hunk 4]
```diff
@@ -29,85 +33,112 @@ class RunbotBuild(models.Model):
...
 
     @api.multi
     def _branch_name(self):
-        if self.branch_id.name.find("pull") >= 0:
-            branch = _(u"PR #{}").format(self.branch_id.branch_name)
-        else:
-            branch = self.branch_id.branch_name
-        self.branch_name = branch
+        for record in self:
+            if record.branch_id.name.find("pull") >= 0:
```
Comment (moylop260) [inline runbot_send_email/models/runbot_build.py]: Use `if 'pull' in record.branch_id.name:`

File: runbot_send_email/models/runbot_build.py [hunk 5]
```diff
@@ -145,22 +176,26 @@ def action_send_email(self):
...
+            if partner not in record.message_partner_ids:
+                record.message_subscribe([partner.id])
+            email_act = record.action_send_email()
+            if email_act and email_act.get('context'):
+                email_ctx = email_act['context']
+                record.with_context(email_ctx).message_post_with_template(
+                    email_ctx.get('default_template_id'))
+                _logger.info('Sent email to: %s, Build: %s', email_to,
+                             name_build)
+            return True
```
Comment (moylop260) [inline runbot_send_email/models/runbot_build.py]: You have a `return True` inside of `for`
You can remove it

**Pull request merged**